docs/installation/certificate.asciidoc

```diff
@@ -0,0 +1,319 @@
+Before addressing the implementation and management of SSL/TLS certificates, it is important to understand the basic concepts related to security and certificates. The following definitions will be used throughout this documentation:
+
+- SSL/TLS: Secure Sockets Layer/Transport Layer Security, a security protocol that allows encrypting communications between a client and a server.
+- SSL/TLS certificate: an electronic file that contains information to verify the identity of a server and establish a secure connection.
+- Certificate Authority (CA): an entity that issues and manages SSL/TLS certificates by verifying the identity of the certificate owner.
+- Intermediate Certificate: a type of digital certificate that is issued by a trusted root certificate authority and is used to establish a chain of trust between the root certificate and end-entity certificates.
+- Private key: an encryption key used to protect confidential information, known only to the certificate owner.
+- Public key: an encryption key used to decrypt information encrypted using the private key, known to all users.
+
+By understanding these basic concepts, you will be better tooled to understand and implement SSL/TLS certificates for the captive web portal and RADIUS.
+
+==== Important note before starting
+
+This Documentation concern PacketFence V11.2 and later
+
+Wildcard certificate is strictly restricted to HTTP, you can't use this type of certificate for RADIUS. 
+If you plan to implement certificate for HTTP and RADIUS, we recommand you to use only one certificate to facilitate management of these.
+
+=== Implementation of SSL/TLS Certificate for Captive Web Portal (HTTP)
+
+==== Generate a Certificate Signing Request (CSR)
+
+To implement an SSL/TLS certificate for the captive web portal (HTTP), the first step is to generate a Certificate Signing Request (CSR). The CSR includes information about the organization requesting the certificate, the domain name of the captive portal, and the private key that will be used to encrypt communications.
+
+- Log on the admin web interface (GUI)
+
+- Go to Configuration > System Configuration > SSL Certificates
+
+image::certificate/14-HTTP-CSR.png[scaledwidth="100%",alt="CSR"]
+
+- Click on “Generate Signing Request (CSR)
+
+- Complete the following using your own information, Common Name must match the captive portal hostname in Configuration > General Configuration > Hostname
+
+image::certificate/16-CSR.png[scaledwidth="100%",alt="CSR"]
+
+Warning: This capture have been made on PacketFence V13. If you are using a lower version (not under V11.2) Subject Alternative Names will be automatically generated from Common Name field.
+
+- Save the CSR and private key to a secure location, you will need it to renew your certificate.
+
+- You can also generate the CSR with CLI using:
+
+[source, shell]
+----
+openssl req -newkey rsa:2048 -keyout PRIVATEKEY.key -out MYCSR.csr
+----
+
+==== Submit the CSR to a Certificate Authority (CA)
+
```

Comment:
I suggest to simplify this section. PacketFence is only use to generate CSR, nothing more.

Reply:
Lets talk about it before merge.